zp-relayer/utils/helpers.ts

```diff
@@ -174,6 +174,10 @@ export function waitForFunds(web3: Web3, address: string, cb: (balance: BN) => v
         retry(new Error('Not enough balance'))
       }
     },
-    { forever: true, factor: 1 }
+    {
+      forever: true,
+      factor: 1,
+      maxTimeout: 60000,
```

Comment:
Should we also add `minTimeout: 60000` ?

Reply:
Yeah, missed it